daemon/web/components/TerminalView/index.sass

```diff
@@ -2,14 +2,16 @@
 
 .terminal-view
   box-sizing: border-box
-  height: 100%
+  height: 496px
+  width: &-width
+  vertical-align: middle
 
   > .text-area
     background: $darkgrey
     color: $white
```

Comment:
`background`, `color` can both be utility classes as well (they might already exist, just add them to `decoration.sass` if not)